frame/staking/src/pallet/mod.rs

```diff
@@ -1148,8 +1143,18 @@ pub mod pallet {
 			ensure!(ledger.active >= MinNominatorBond::<T>::get(), Error::<T>::InsufficientBond);
 			let stash = &ledger.stash;
 
+			let is_nominator = Nominators::<T>::contains_key(stash);
+
+			// If the nominator is not decodable - throw an error. The only reason for that could be
+			// a decrease of `MaxNominatorsCount`, which should be accompanied by a migration that
```

Comment:
This comment is wrong -- You probably meant to say `MaxNominations`.

Reply:
Yes, a typo!